apps/hash-ai-worker-ts/src/activities/flow-activities/web-search-action.ts

```diff
@@ -18,6 +18,13 @@ export const webSearchAction: FlowActionActivity = async ({ inputs }) => {
 
   const webPagesUrls = webSearchResults
     .map(({ url }) => url)
+    /**
+     * The coordinator agent using this method does not have the ability to directly
+     * interact with PDFs, so we filter these out for now.
+     *
+     * @todo: account for PDFs being returned in search results in the coordinator agent
```

Comment:
This seems worth addressing / not overlooking and we probably won't touch this function much, can you create a task for it if you haven't already please, and put it in next sprint?

Reply:
There's an existing ticket, have linked it in the comment in https://github.com/hashintel/hash/pull/4399/commits/13a747564972dd57710a742a6fa008847139f3b5